JapaneseFilter/sections/css_extended.txt

```diff
@@ -7,6 +7,9 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/104224#issuecomment-1002865063
 ! TODO: replace the rule below after the release of the extension 4.0
+! https://github.com/AdguardTeam/AdguardFilters/issues/114765
+full-count.jp#?#.l-sidebar-ads > li:if-not(> a[href^="https://full-count.jp"])
+full-count.jp#?#.l-sidebar > aside.l-sidebar__block:has(> div.l-sidebar-ranking > h2:contains(SAMURAI JAPAN))
```

Comment:
Wrong position - between comment and the rule of another issue.
Move the first below, the second(recommendation) - to annoyances.